apps/dashboard/src/components/conditions-editor/rule-actions.tsx

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from 'react';
+import { isRuleGroup, ActionWithRulesProps, getParentPath } from 'react-querybuilder';
+import { RiMore2Fill } from 'react-icons/ri';
+
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from '@/components/primitives/dropdown-menu';
+import { useConditionsEditorContext } from './conditions-editor-context';
+import { CompactButton } from '@/components/primitives/button-compact';
+import { Delete } from '@/components/icons/delete';
+import { SquareTwoStack } from '@/components/icons/square-two-stack';
+
+export const RuleActions = ({ path, ruleOrGroup }: ActionWithRulesProps) => {
```

Comment:
This component represents the group or conditions actions menu `...`. The duplicate action should be disabled when there are 10 conditions applied in the group.

![Screenshot 2025-01-14 at 12 33 32](https://github.com/user-attachments/assets/46976060-be37-471a-90e5-07744197db7b)